tests/tools_test.py

```diff
@@ -126,3 +136,28 @@ def test_combined_curves():
             ],
         ),
     )
+
+
+@pytest.mark.filterwarnings("ignore:No suitable")
+def test_orso_datasets_tool():
+    def normalized_ioq(filename: Filename[SampleRun]) -> ReflectivityOverQ:
+        return filename
+
+    def orso_dataset(filename: Filename[SampleRun]) -> OrsoIofQDataset:
+        class Reduction:
+            corrections = []  # noqa: RUF012
+
+        return OrsoDataset(
+            Orso({}, Reduction, [], name=f'{filename}.orso'), np.ones((0, 0))
+        )
+
+    workflow = sl.Pipeline(
+        [normalized_ioq, orso_dataset], params={Filename[SampleRun]: 'default'}
+    )
+    datasets = orso_datasets_from_measurements(
+        workflow,
+        [{}, {Filename[SampleRun]: 'special'}],
+        scale_to_overlap=False,
+    )
+    assert len(datasets) == 2
+    assert tuple(d.info.name for d in datasets) == ('default.orso', 'special.orso')
```

Comment:
Above you said `orso_dataset_from_measurements` should be able to perform/record possibly distinct corrections applied for each run. Here you are testing the same correction, but it retrieves a different value (the filename). Is that the same?

Reply:
This doesn't test any aspect of the "correction finder"-mechanism, so I'm not sure what you mean.
This only tests if the provided parameter value was used, or if the default parameter value was used.